framework/src/executioners/EigenExecutionerBase.C

```diff
@@ -240,6 +249,8 @@ EigenExecutionerBase::inversePowerIteration(unsigned int min_iter,
     _problem.getAuxiliarySystem().copyOldSolutions();
     if ( _problem.getDisplacedProblem() != NULL )
     {
+      _problem.getDisplacedProblem()->nlSys().update();
```

Comment:
The reason we need these two lines is that in `FEProblem::solve()`, there is only `syncSolutions()`, but not the updates. I guess the more proper fix will be immediately performing update in `DisplaceProblem::syncSolutions()`. How does this sounds?